sdk/samples/storage/blobs/src/blobs_client_example.c

```diff
@@ -91,14 +91,23 @@ int main()
 
   // This validation is only for the first time SDK client is used. API will return not implemented
   // if samples were built with no_http lib.
-  if (blob_upload_result == AZ_ERROR_DEPENDENCY_NOT_PROVIDED)
+  if (blob_upload_result == AZ_ERROR_DEPENDENCY_NOT_PROVIDED_HTTP)
   {
     printf("Running sample with no_op HTTP implementation.\nRecompile az_core with an HTTP client "
            "implementation like CURL to see sample sending network requests.\n\n"
            "i.e. cmake -DTRANSPORT_CURL=ON ..\n\n");
 
     return 1;
   }
+  else if (blob_upload_result == AZ_ERROR_DEPENDENCY_NOT_PROVIDED_PAL)
+  {
+    printf("Running sample with no_op PAL implementation.\nRecompile az_core with a PAL "
```

Comment:
avoid PAL, use platform. Please search our code base for PAL, and correct all instances, if there are more.